src/IIIFPresentation/API/Features/Storage/Validators/FlatCollectionValidator.cs

```diff
@@ -9,18 +9,33 @@ public FlatCollectionValidator()
     {
         RuleSet("create", () =>
         {
-            RuleFor(a => a.Created).Empty().WithMessage("Created cannot be set");
-            RuleFor(a => a.Modified).Empty().WithMessage("Modified cannot be set");
-            RuleFor(a => a.Id).Empty().WithMessage("Id cannot be set");
-            RuleFor(a => a.Parent).NotEmpty().WithMessage("Creating a new collection requires a parent");
+            RuleFor(f => f.Created).Empty().WithMessage("Created cannot be set");
+            RuleFor(f => f.Modified).Empty().WithMessage("Modified cannot be set");
+            RuleFor(f => f.Id).Empty().WithMessage("Id cannot be set");
+            RuleFor(f => f.Parent).NotEmpty().WithMessage("Creating a new collection requires a parent");
         });
         
         RuleSet("update", () =>
         {
-            RuleFor(a => a.Created).Empty().WithMessage("Created cannot be set");
-            RuleFor(a => a.Modified).Empty().WithMessage("Modified cannot be set");
-            RuleFor(a => a.Id).Empty().WithMessage("Id cannot be set");
-            RuleFor(a => a.Parent).NotEmpty().WithMessage("Updating a collection requires a parent to be set");
+            RuleFor(f => f.Created).Empty().WithMessage("Created cannot be set");
+            RuleFor(f => f.Modified).Empty().WithMessage("Modified cannot be set");
+            RuleFor(f => f.Id).Empty().WithMessage("Id cannot be set");
+            RuleFor(f => f.Parent).NotEmpty().WithMessage("Updating a collection requires a parent to be set");
+            RuleFor(f => f.Context).Empty().WithMessage("Context cannot be set");
+            RuleFor(f => f.Items).Empty().WithMessage("Items cannot be set");
+            RuleFor(f => f.Tags).Empty().WithMessage("Tags cannot be set");
+            RuleFor(f => f.Thumbnail).Empty().WithMessage("Thumbnail cannot be set");
+            RuleFor(f => f.Type).Empty().WithMessage("Type cannot be set");
+            RuleFor(f => f.View).Empty().WithMessage("View cannot be set");
+            RuleFor(f => f.ItemsOrder).Empty().WithMessage("Items order cannot be set");
+            RuleFor(f => f.CreatedBy).Empty().WithMessage("Created by cannot be set");
+            RuleFor(f => f.ModifiedBy).Empty().WithMessage("Modified By cannot be set");
+            RuleFor(f => f.PartOf).Empty().WithMessage("Part of cannot be set");
```

Comment:
Would these responses be slightly easier to read if the properties were enclosed in single quotes? e.g
> 'Modified by' cannot be set